tools/runners/run-test-suite.sh

```diff
@@ -59,6 +59,25 @@ fi
 
 rm -f $TEST_FAILED $TEST_PASSED
 
+ROOT_DIR=""
+CURRENT_DIR=`pwd`
+PATH_STEP=2
+while true
+do
+    TMP_ROOT_DIR=`(echo $CURRENT_DIR; echo $0; echo $ENGINE; echo $TESTS) | cut -f1-$PATH_STEP -d/ | uniq -d`
```

Comment:
I haven't checked it yet. I'll check it later. I use symbolic links in my working paths. My prediction is that we need `pwd -P`.